### ISSUE TITLE
[Bug] Match giving response 200 in v0.10.0 but 250 in v0.9.0

### ISSUE THREAD

**Issue by TechieDylan** [nuxencs/seasonpackarr#144]
### Version

v0.10.0

### Description

I have gotten a clear match in v0.10.0 but noticed it was never being injected upon looking ive found that v0.10.0 is returning code 200 for match and non-match.

confirmed in v0.9.0 that the same match give response code 250 and non-match 200 as expected

### Expected Behavior

I'm expecting response code 250 on match based on the documents and previous versions

### Relevant log output

```shell
Match in v0.9.0

2024-09-30T23:50:57.108615412Z {"level":"info","module":"processor","time":"2024-09-30T23:50:57Z","message":"starting to process season pack request"}
2024-09-30T23:50:57.108735025Z {"level":"info","module":"processor","time":"2024-09-30T23:50:57Z","message":"no clientname defined. trying to use default client"}
2024-09-30T23:50:57.108811188Z {"level":"info","module":"processor","release":"Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)","clientname":"default","time":"2024-09-30T23:50:57Z","message":"using default client serving at 192.168.11.11:8079"}
2024-09-30T23:51:00.831102942Z {"level":"debug","module":"processor","release":"Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)","clientname":"default","time":"2024-09-30T23:51:00Z","message":"formatted season pack name: Richard.Hammond's.Workshop.(2021).S04.(1080p.AMZN.WEB-DL.H265.SDR.DDP2.0.English-DarQ)"}
2024-09-30T23:51:00.859957151Z {"level":"debug","module":"processor","release":"Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)","clientname":"default","time":"2024-09-30T23:51:00Z","message":"matched torrent from client: name(Richard Hammond's Workshop (2021) S04E05 Pedal to the Metal (1080p AMZN WEB-DL H264 SDR DDP 2.0 English-DarQ).mkv), size(3056411480), hash(a7a045707ae97f0cdd3883662ffc3a96706574d7)"}
2024-09-30T23:51:00.861977724Z {"level":"debug","module":"processor","release":"Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)","clientname":"default","time":"2024-09-30T23:51:00Z","message":"matched torrent from client: name(Richard Hammond's Workshop (2021) S04E06 Deadline for D-Day (1080p AMZN WEB-DL H265 SDR DDP 2.0 English-DarQ).mkv), size(1466037115), hash(4a5c5da4f7274a40fd967b8f4bc00338ed0c2d8b)"}
2024-09-30T23:51:00.863930953Z {"level":"debug","module":"processor","release":"Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)","clientname":"default","time":"2024-09-30T23:51:00Z","message":"matched torrent from client: name(Richard Hammond's Workshop (2021) S04E06 Deadline for D-Day (1080p AMZN WEB-DL H265 SDR DDP 2.0 English-DarQ).mkv), size(1466037115), hash(65fb112db5395f317f5035aa783597be2215b280)"}
2024-09-30T23:51:00.865954059Z {"level":"debug","module":"processor","release":"Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)","clientname":"default","time":"2024-09-30T23:51:00Z","message":"matched torrent from client: name(Richard Hammond's Workshop (2021) S04E05 Pedal to the Metal (1080p AMZN WEB-DL H265 SDR DDP 2.0 English-DarQ).mkv), size(1649566707), hash(3bccd6d67704cf73dc0e2f31168519acdf7710ec)"}
2024-09-30T23:51:00.867987320Z {"level":"debug","module":"processor","release":"Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)","clientname":"default","time":"2024-09-30T23:51:00Z","message":"matched torrent from client: name(Richard Hammond's Workshop (2021) S04E01 Something Has to Change (1080p AMZN WEB-DL H265 SDR DDP 2.0 English-DarQ).mkv), size(1598246977), hash(b6dddde32d4125731d5216e505c54902515b754d)"}
2024-09-30T23:51:00.871492003Z {"level":"debug","module":"processor","release":"Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)","clientname":"default","time":"2024-09-30T23:51:00Z","message":"matched torrent from client: name(Richard Hammond's Workshop (2021) S04E01 Something Has to Change (1080p AMZN WEB-DL H265 SDR DDP 2.0 English-DarQ).mkv), size(1598246977), hash(53b86576e06f86abce5965d8feb97d147564c637)"}
2024-09-30T23:51:00.873478073Z {"level":"debug","module":"processor","release":"Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)","clientname":"default","time":"2024-09-30T23:51:00Z","message":"matched torrent from client: name(Richard Hammond's Workshop (2021) S04E05 Pedal to the Metal (1080p AMZN WEB-DL H264 SDR DDP 2.0 English-DarQ).mkv), size(3056411480), hash(80f905289d1666a3673172152ee7504e6732806a)"}
2024-09-30T23:51:00.875308849Z {"level":"debug","module":"processor","release":"Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)","clientname":"default","time":"2024-09-30T23:51:00Z","message":"matched torrent from client: name(Richard Hammond's Workshop (2021) S04E05 Pedal to the Metal (1080p AMZN WEB-DL H264 SDR DDP 2.0 English-DarQ).mkv), size(3056411480), hash(f64317c4f4389533e2daebbf98b0c25699b01467)"}
2024-09-30T23:51:00.877055715Z {"level":"debug","module":"processor","release":"Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)","clientname":"default","time":"2024-09-30T23:51:00Z","message":"matched torrent from client: name(Richard Hammond's Workshop (2021) S04E03 Testing Times (1080p AMZN WEB-DL H265 SDR DDP 2.0 English-DarQ).mkv), size(1435946759), hash(459a5190dcfab8f198e6dc4548394a91b91a1c37)"}
2024-09-30T23:51:00.878679240Z {"level":"debug","module":"processor","release":"Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)","clientname":"default","time":"2024-09-30T23:51:00Z","message":"matched torrent from client: name(Richard Hammond's Workshop (2021) S04E04 Episode 4 (1080p AMZN WEB-DL H265 SDR DDP 2.0 English-DarQ).mkv), size(1093505159), hash(cf77524360d60d4d63ffb6b224c31971b5ccdf54)"}
2024-09-30T23:51:00.880463152Z {"level":"debug","module":"processor","release":"Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)","clientname":"default","time":"2024-09-30T23:51:00Z","message":"matched torrent from client: name(Richard Hammond's Workshop (2021) S04E06 Deadline for D-Day (1080p AMZN WEB-DL H265 SDR DDP 2.0 English-DarQ).mkv), size(1466037115), hash(dfe342db8e666abe7adb2610cf2f117a0f40d191)"}
2024-09-30T23:51:00.882289573Z {"level":"debug","module":"processor","release":"Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)","clientname":"default","time":"2024-09-30T23:51:00Z","message":"matched torrent from client: name(Richard Hammond's Workshop (2021) S04E05 Pedal to the Metal (1080p AMZN WEB-DL H265 SDR DDP 2.0 English-DarQ).mkv), size(1649566707), hash(aba26b0f91eeb26ed6747417a5fe983d17c9cf51)"}
2024-09-30T23:51:00.884462591Z {"level":"debug","module":"processor","release":"Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)","clientname":"default","time":"2024-09-30T23:51:00Z","message":"matched torrent from client: name(Richard Hammond's Workshop (2021) S04E06 Deadline for D-Day (1080p AMZN WEB-DL H264 SDR DDP 2.0 English-DarQ).mkv), size(3055697493), hash(f7220279e26a9e731a8a92a35b2b10a1c2e2acd5)"}
2024-09-30T23:51:00.886282299Z {"level":"debug","module":"processor","release":"Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)","clientname":"default","time":"2024-09-30T23:51:00Z","message":"matched torrent from client: name(Richard Hammond's Workshop (2021) S04E03 Testing Times (1080p AMZN WEB-DL H265 SDR DDP 2.0 English-DarQ).mkv), size(1435946759), hash(3a18f565d6596b9b8573241cba75b5a3cecdc84f)"}
2024-09-30T23:51:00.888058312Z {"level":"debug","module":"processor","release":"Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)","clientname":"default","time":"2024-09-30T23:51:00Z","message":"matched torrent from client: name(Richard Hammond's Workshop (2021) S04E02 A New Era (1080p AMZN WEB-DL H265 SDR DDP 2.0 English-DarQ).mkv), size(1304936214), hash(8c693a8c33ea5e03a18da483fb8669cd7df3d105)"}
2024-09-30T23:51:00.889614834Z {"level":"debug","module":"processor","release":"Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)","clientname":"default","time":"2024-09-30T23:51:00Z","message":"matched torrent from client: name(Richard Hammond's Workshop (2021) S04E04 Episode 4 (1080p AMZN WEB-DL H265 SDR DDP 2.0 English-DarQ).mkv), size(1093505159), hash(342319e7d1b93917d04003d03ab163bd50412d87)"}
2024-09-30T23:51:00.891416080Z {"level":"debug","module":"processor","release":"Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)","clientname":"default","time":"2024-09-30T23:51:00Z","message":"matched torrent from client: name(Richard Hammond's Workshop (2021) S04E06 Deadline for D-Day (1080p AMZN WEB-DL H264 SDR DDP 2.0 English-DarQ).mkv), size(3055697493), hash(89a7d4cd94603c8d2d3d1118094b63b7148a4195)"}
2024-09-30T23:51:00.893174408Z {"level":"debug","module":"processor","release":"Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)","clientname":"default","time":"2024-09-30T23:51:00Z","message":"matched torrent from client: name(Richard Hammond's Workshop (2021) S04E06 Deadline for D-Day (1080p AMZN WEB-DL H264 SDR DDP 2.0 English-DarQ).mkv), size(3055697493), hash(5ec4fde2f5a153a70e52788a1cd263fe5eabd4f3)"}
2024-09-30T23:51:00.894746448Z {"level":"debug","module":"processor","release":"Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)","clientname":"default","time":"2024-09-30T23:51:00Z","message":"matched torrent from client: name(Richard Hammond's Workshop (2021) S04E04 Episode 4 (1080p AMZN WEB-DL H265 SDR DDP 2.0 English-DarQ).mkv), size(1093505159), hash(295ee0a6b7b02b4703a6f8d5aefd6628216f23a8)"}
2024-09-30T23:51:00.896545325Z {"level":"debug","module":"processor","release":"Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)","clientname":"default","time":"2024-09-30T23:51:00Z","message":"matched torrent from client: name(Richard Hammond's Workshop (2021) S04E02 A New Era (1080p AMZN WEB-DL H265 SDR DDP 2.0 English-DarQ).mkv), size(1304936214), hash(d6a7ca5b214dfaa3c3bcc271937feacb40b25aef)"}
2024-09-30T23:51:00.898491064Z {"level":"debug","module":"processor","release":"Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)","clientname":"default","time":"2024-09-30T23:51:00Z","message":"matched torrent from client: name(Richard Hammond's Workshop (2021) S04E05 Pedal to the Metal (1080p AMZN WEB-DL H265 SDR DDP 2.0 English-DarQ).mkv), size(1649566707), hash(6e4072a978fa425e9bb74ce8958d7024f3ece922)"}
2024-09-30T23:51:00.901700494Z {"level":"debug","module":"processor","release":"Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)","clientname":"default","time":"2024-09-30T23:51:00Z","message":"matched torrent from client: name(Richard Hammond's Workshop (2021) S04E03 Testing Times (1080p AMZN WEB-DL H265 SDR DDP 2.0 English-DarQ).mkv), size(1435946759), hash(ca551c01dc700067836bb1bcd140d45d1033204c)"}
2024-09-30T23:51:00.903668298Z {"level":"debug","module":"processor","release":"Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)","clientname":"default","time":"2024-09-30T23:51:00Z","message":"matched torrent from client: name(Richard Hammond's Workshop (2021) S04E02 A New Era (1080p AMZN WEB-DL H265 SDR DDP 2.0 English-DarQ).mkv), size(1304936214), hash(0e5e9d523cfe7054825f8eacdeecccfe85526951)"}
2024-09-30T23:51:00.905541697Z {"level":"debug","module":"processor","release":"Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)","clientname":"default","time":"2024-09-30T23:51:00Z","message":"matched torrent from client: name(Richard Hammond's Workshop (2021) S04E01 Something Has to Change (1080p AMZN WEB-DL H265 SDR DDP 2.0 English-DarQ).mkv), size(1598246977), hash(6e5b04b00e670ea994296c8620e7759402143c10)"}
2024-09-30T23:51:01.145024521Z {"level":"warn","sender":"discord","time":"2024-09-30T23:51:01Z","message":"no webhook defined, skipping notification"}
2024-09-30T23:51:01.145082628Z {"level":"info","module":"processor","release":"Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)","clientname":"default","time":"2024-09-30T23:51:01Z","message":"successfully matched season pack to episodes in client"}
2024-09-30T23:51:01.145023530Z 2024/09/30 23:51:01 [7c33969f356d/NDCPSmUmDP-000044] "POST http://127.0.0.1:42069/api/pack HTTP/1.1" from 127.0.0.1:37338 - 250 0B in 4.036771959s

_______________________________________________________________

Match in v0.10.0

2024-09-30T23:54:16Z INF starting to process season pack request module=processor
2024-09-30T23:54:16Z INF no clientname defined. trying to use default client module=processor
2024-09-30T23:54:16Z INF using default client serving at 192.168.11.11:8079 clientname=default module=processor release="Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)"
2024-09-30T23:54:26Z DBG formatted season pack name: Richard.Hammond's.Workshop.(2021).S04.(1080p.AMZN.WEB-DL.H265.SDR.DDP2.0.English-DarQ) clientname=default module=processor release="Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)"
2024-09-30T23:54:26Z DBG matched torrent from client: name(Richard Hammond's Workshop (2021) S04E05 Pedal to the Metal (1080p AMZN WEB-DL H264 SDR DDP 2.0 English-DarQ).mkv), size(3056411480), hash(a7a045707ae97f0cdd3883662ffc3a96706574d7) clientname=default module=processor release="Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)"
2024-09-30T23:54:26Z DBG matched torrent from client: name(Richard Hammond's Workshop (2021) S04E06 Deadline for D-Day (1080p AMZN WEB-DL H265 SDR DDP 2.0 English-DarQ).mkv), size(1466037115), hash(4a5c5da4f7274a40fd967b8f4bc00338ed0c2d8b) clientname=default module=processor release="Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)"
2024-09-30T23:54:26Z DBG matched torrent from client: name(Richard Hammond's Workshop (2021) S04E06 Deadline for D-Day (1080p AMZN WEB-DL H265 SDR DDP 2.0 English-DarQ).mkv), size(1466037115), hash(65fb112db5395f317f5035aa783597be2215b280) clientname=default module=processor release="Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)"
2024-09-30T23:54:26Z DBG matched torrent from client: name(Richard Hammond's Workshop (2021) S04E05 Pedal to the Metal (1080p AMZN WEB-DL H265 SDR DDP 2.0 English-DarQ).mkv), size(1649566707), hash(3bccd6d67704cf73dc0e2f31168519acdf7710ec) clientname=default module=processor release="Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)"
2024-09-30T23:54:26Z DBG matched torrent from client: name(Richard Hammond's Workshop (2021) S04E01 Something Has to Change (1080p AMZN WEB-DL H265 SDR DDP 2.0 English-DarQ).mkv), size(1598246977), hash(b6dddde32d4125731d5216e505c54902515b754d) clientname=default module=processor release="Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)"
2024-09-30T23:54:26Z DBG matched torrent from client: name(Richard Hammond's Workshop (2021) S04E01 Something Has to Change (1080p AMZN WEB-DL H265 SDR DDP 2.0 English-DarQ).mkv), size(1598246977), hash(53b86576e06f86abce5965d8feb97d147564c637) clientname=default module=processor release="Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)"
2024-09-30T23:54:26Z DBG matched torrent from client: name(Richard Hammond's Workshop (2021) S04E05 Pedal to the Metal (1080p AMZN WEB-DL H264 SDR DDP 2.0 English-DarQ).mkv), size(3056411480), hash(80f905289d1666a3673172152ee7504e6732806a) clientname=default module=processor release="Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)"
2024-09-30T23:54:26Z DBG matched torrent from client: name(Richard Hammond's Workshop (2021) S04E05 Pedal to the Metal (1080p AMZN WEB-DL H264 SDR DDP 2.0 English-DarQ).mkv), size(3056411480), hash(f64317c4f4389533e2daebbf98b0c25699b01467) clientname=default module=processor release="Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)"
2024-09-30T23:54:26Z DBG matched torrent from client: name(Richard Hammond's Workshop (2021) S04E03 Testing Times (1080p AMZN WEB-DL H265 SDR DDP 2.0 English-DarQ).mkv), size(1435946759), hash(459a5190dcfab8f198e6dc4548394a91b91a1c37) clientname=default module=processor release="Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)"
2024-09-30T23:54:26Z DBG matched torrent from client: name(Richard Hammond's Workshop (2021) S04E04 Episode 4 (1080p AMZN WEB-DL H265 SDR DDP 2.0 English-DarQ).mkv), size(1093505159), hash(cf77524360d60d4d63ffb6b224c31971b5ccdf54) clientname=default module=processor release="Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)"
2024-09-30T23:54:26Z DBG matched torrent from client: name(Richard Hammond's Workshop (2021) S04E06 Deadline for D-Day (1080p AMZN WEB-DL H265 SDR DDP 2.0 English-DarQ).mkv), size(1466037115), hash(dfe342db8e666abe7adb2610cf2f117a0f40d191) clientname=default module=processor release="Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)"
2024-09-30T23:54:26Z DBG matched torrent from client: name(Richard Hammond's Workshop (2021) S04E05 Pedal to the Metal (1080p AMZN WEB-DL H265 SDR DDP 2.0 English-DarQ).mkv), size(1649566707), hash(aba26b0f91eeb26ed6747417a5fe983d17c9cf51) clientname=default module=processor release="Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)"
2024-09-30T23:54:26Z DBG matched torrent from client: name(Richard Hammond's Workshop (2021) S04E06 Deadline for D-Day (1080p AMZN WEB-DL H264 SDR DDP 2.0 English-DarQ).mkv), size(3055697493), hash(f7220279e26a9e731a8a92a35b2b10a1c2e2acd5) clientname=default module=processor release="Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)"
2024-09-30T23:54:26Z DBG matched torrent from client: name(Richard Hammond's Workshop (2021) S04E03 Testing Times (1080p AMZN WEB-DL H265 SDR DDP 2.0 English-DarQ).mkv), size(1435946759), hash(3a18f565d6596b9b8573241cba75b5a3cecdc84f) clientname=default module=processor release="Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)"
2024-09-30T23:54:26Z DBG matched torrent from client: name(Richard Hammond's Workshop (2021) S04E02 A New Era (1080p AMZN WEB-DL H265 SDR DDP 2.0 English-DarQ).mkv), size(1304936214), hash(8c693a8c33ea5e03a18da483fb8669cd7df3d105) clientname=default module=processor release="Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)"
2024-09-30T23:54:26Z DBG matched torrent from client: name(Richard Hammond's Workshop (2021) S04E04 Episode 4 (1080p AMZN WEB-DL H265 SDR DDP 2.0 English-DarQ).mkv), size(1093505159), hash(342319e7d1b93917d04003d03ab163bd50412d87) clientname=default module=processor release="Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)"
2024-09-30T23:54:26Z DBG matched torrent from client: name(Richard Hammond's Workshop (2021) S04E06 Deadline for D-Day (1080p AMZN WEB-DL H264 SDR DDP 2.0 English-DarQ).mkv), size(3055697493), hash(89a7d4cd94603c8d2d3d1118094b63b7148a4195) clientname=default module=processor release="Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)"
2024-09-30T23:54:26Z DBG matched torrent from client: name(Richard Hammond's Workshop (2021) S04E06 Deadline for D-Day (1080p AMZN WEB-DL H264 SDR DDP 2.0 English-DarQ).mkv), size(3055697493), hash(5ec4fde2f5a153a70e52788a1cd263fe5eabd4f3) clientname=default module=processor release="Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)"
2024-09-30T23:54:26Z DBG matched torrent from client: name(Richard Hammond's Workshop (2021) S04E04 Episode 4 (1080p AMZN WEB-DL H265 SDR DDP 2.0 English-DarQ).mkv), size(1093505159), hash(295ee0a6b7b02b4703a6f8d5aefd6628216f23a8) clientname=default module=processor release="Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)"
2024-09-30T23:54:26Z DBG matched torrent from client: name(Richard Hammond's Workshop (2021) S04E02 A New Era (1080p AMZN WEB-DL H265 SDR DDP 2.0 English-DarQ).mkv), size(1304936214), hash(d6a7ca5b214dfaa3c3bcc271937feacb40b25aef) clientname=default module=processor release="Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)"
2024-09-30T23:54:26Z DBG matched torrent from client: name(Richard Hammond's Workshop (2021) S04E05 Pedal to the Metal (1080p AMZN WEB-DL H265 SDR DDP 2.0 English-DarQ).mkv), size(1649566707), hash(6e4072a978fa425e9bb74ce8958d7024f3ece922) clientname=default module=processor release="Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)"
2024-09-30T23:54:26Z DBG matched torrent from client: name(Richard Hammond's Workshop (2021) S04E03 Testing Times (1080p AMZN WEB-DL H265 SDR DDP 2.0 English-DarQ).mkv), size(1435946759), hash(ca551c01dc700067836bb1bcd140d45d1033204c) clientname=default module=processor release="Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)"
2024-09-30T23:54:26Z DBG matched torrent from client: name(Richard Hammond's Workshop (2021) S04E02 A New Era (1080p AMZN WEB-DL H265 SDR DDP 2.0 English-DarQ).mkv), size(1304936214), hash(0e5e9d523cfe7054825f8eacdeecccfe85526951) clientname=default module=processor release="Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)"
2024-09-30T23:54:26Z DBG matched torrent from client: name(Richard Hammond's Workshop (2021) S04E01 Something Has to Change (1080p AMZN WEB-DL H265 SDR DDP 2.0 English-DarQ).mkv), size(1598246977), hash(6e5b04b00e670ea994296c8620e7759402143c10) clientname=default module=processor release="Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)"
2024-09-30T23:54:27Z DBG no webhook defined, skipping notification sender=discord
2024-09-30T23:54:27Z INF successfully matched season pack to episodes in client clientname=default module=processor release="Richard Hammond's Workshop (2021) S04 (1080p AMZN WEB-DL H265 SDR DDP 2 0 English-DarQ)"
2024-09-30T23:54:27Z TRC incoming_request bytes_in=128 bytes_out=-1 latency_ms=10687.043279 method=POST proto=HTTP/1.1 remote_ip=127.0.0.1 status=200 type=access url=/api/pack user_agent=Go-http-client/1.1
```

### Screenshots

v0.9.0 response code for that match

<img width="1014" alt="v0 9 0 - working" src="https://github.com/user-attachments/assets/b7bf7dbf-4d04-4829-9c1f-64ff27ea41c6">

__________________________________

v0.10.0 response code for the same match

<img width="1183" alt="v0 10 0 - not working" src="https://github.com/user-attachments/assets/90efc0fd-34ef-4031-abe0-2f66bba2e8fc">

**Comment by nuxencs** [comment]
thanks for pointing this out, looking at it right now 👍🏼

**Comment by nuxencs** [comment]
@TechieDylan this is fixed in v0.10.1

**Comment by TechieDylan** [comment]
Excellent thank you, confirmed it's working on my end as well
